src/test/kotlin/no/nav/syfo/behandlerdialog/AvvistMeldingSpek.kt

```diff
@@ -86,11 +84,12 @@ class AvvistMeldingSpek : Spek({
             }
             it("stores avvist melding from kafka as oppgave in database and publish as new oppgave also when ubesvartoppgave exists for same referanseUuid") {
                 val referanseUuid = UUID.randomUUID()
+                val paaminnelsesOppgaveUUID = UUID.randomUUID()
                 database.connection.use { connection ->
                     connection.createPersonOppgave(
                         personoppgave = PersonOppgave(
```

Comment:
Skal finnes generator-utils for dette også, slik at testene ikke blir så lange. Men det er bare pirk, da 😄